metadata_fetcher/fetchers/Fetcher.py

```diff
@@ -62,8 +71,12 @@ def fetchtos3(self, page):
 
     def fetch_page(self):
         page = self.build_fetch_request()
-        response = requests.get(**page)
-        response.raise_for_status()
+        try:
+            response = requests.get(**page)
+            response.raise_for_status()
+        except requests.exceptions.HTTPError:
+            raise FetchError(
+                f"[{self.collection_id}]: unable to fetch page {page}")
```

Comment:
Would it be helpful to include the error message from requests in the output to give some insight into what went wrong?